samples/unit-tests/svgrenderer/text/demo.js

```diff
@@ -864,6 +864,26 @@ QUnit.test('Text height', function (assert) {
             })
             .add();
 
+        // Backwards compatibility, #22807
+        assert.deepEqual(
+            renderer.fontMetrics(14),
+            {
+                h: 17,
+                b: 14,
+                f: 14
```

Comment:
Hmmmm, this errors locally for me, but its just off by a couple 🤔 

Idk how much it matters for karma tests but should we use assert.close instead maybe?

Reply:
Alright!